config/dynamic.c

```diff
@@ -601,7 +601,8 @@ run_dynamic_config(struct gk_config *gk_conf, struct gt_config *gt_conf,
 	if (ret < 0)
 		goto put_gk_gt_config;
 
-	return 0;
+	return change_file_permissions(dy_conf->server_path,
+		get_net_conf()->user, get_net_conf()->mode);
```

Comment:
1. Let's avoid `get_net_conf()` because it means that the Dynamic configuration block now depends on the Net config, so require it as a parameter of `run_dynamic_config()`.

2. The dynamic configuration block should have a `mode` variable to define the mode of its Unix socket, which may not be the same mode of the log file.

3. Use `fchmod()` as explained on another comment to change the mode.